modmesh/app/euler1d.py

```diff
@@ -104,16 +104,16 @@ def update(self, adata, ndata):
         self.num.figure.canvas.draw()
 
 
-class SolverConfig():
+class GUIConfig:
```

Comment:
Considering that `SolverConfig` and `PlotConfig` have a lot of similar code, I created a parent class to implement the shared parts.

Reply:
I would hope we keep a minimal Python 2/3 compatibility for classes (although I don't think anyone would use Python 2).  That is to make a class derived from `object`:

```python
class GUIConfig(object):
```

In Python 3 it is the same as writing `class GUIConfig:`.  But in Python 2, `class GUIConfig:` is the old-style (pre-2.2) class, and incompatible to the new-style class (2.3 and beyond).

Having the minimal compatibility makes it slight easier for people to copy code to Python 2.